apps/client/src/app/lobby/[roomId]/page.tsx

```diff
@@ -58,6 +58,7 @@ export default function LobbyPage() {
   const startGame = () => {
     if (players.length >= 2) {
       socket.emit('start_game', roomId);
+      redirect(`/game/${roomId}`);
```

Comment:
I don't got redirect here but, received log from server.
Is it suppose to behave like this?

Reply:
changed to router.push   Works better